colorize.plugin.zsh

```diff
@@ -75,13 +75,14 @@ function diff() {
 }
 
 if (( $+commands[grc] )); then
-  function env() {
-    command grc --colour=auto env "$@"
-  }
-
-  function lsblk() {
-    command grc --colour=auto lsblk "$@"
-  }
+  for name in env lsblk as dig gas gcc g++ last ld ifconfig mount mtr netstat ping ping6 ps traceroute; do
+    if (( $+commands[dig] )); then
+      function $name() {
+        command grc --colour=auto $name "$@"
+      }
+    fi
+  done
```

Comment:
This broke all these commands for me:

<img width="414" alt="image" src="https://github.com/zpm-zsh/colorize/assets/8050659/6a04e6f7-ee3e-47e1-a4cd-b44fa98b05b1">

<img width="603" alt="image" src="https://github.com/zpm-zsh/colorize/assets/8050659/34282499-8cec-4203-b3c7-5927c847d4c4">

Reply:
![Uploading image.png…]()